tools/wheel/build-wheels

```diff
@@ -217,7 +216,11 @@ def test_wheel(target, options):
     Runs the test script for the wheel matching the specified target.
     """
     vm = f'cp{target.python_version}'
-    wheel = f'drake-{options.version}-{vm}-{vm}m-manylinux_2_27_x86_64.whl'
+    if target.python_version < '38':
+        abi = 'm'
+    else:
+        abi = ''
+    wheel = f'drake-{options.version}-{vm}-{vm}{abi}-manylinux_2_27_x86_64.whl'
```

Comment:
(Sure would be nice if Reviewable had Suggestions... Or the ability to comment on multiple lines...)

Strictly speaking, I think this:
```suggestion
    if target.python_version < '38':
        abi = f'{vm}m'
    else:
        abi = f'{vm}'
    wheel = f'drake-{options.version}-{vm}-{abi}-manylinux_2_27_x86_64.whl'
```
...might be more accurate? (As a bonus, it makes the `wheel =` line shorter 🙂.)